docs/accessControl.md

```diff
@@ -185,10 +190,16 @@ To block students from viewing closed assessment scores, set `"showClosedAssessm
         "showClosedAssessment": false,
         "showClosedAssessmentScore": false,
         "credit": 100
+    },
+    {
+        "showClosedAssessment": false,
+        "showClosedAssessmentScore": false
     }
 ]
 ```
 
+Note that the `showClosedAssessment` and `showClosedAssessmentScore` statements are listed twice above, once while the assessment is in progress (`startDate` to `endDate`) and once in a second rule that will take effect after the `endDate`.
+
 The `showClosedAssessmentScore` access rule restriction is only useful in conjunction with [disabling real-time grading](assessment.md#disabling-real-time-grading) and setting `"showClosedAssessment": false`.
```

Comment:
This is not your change, but this may not be 100% correct in this case, so piggybacking. Here's a suggestion:
```suggestion
The `showClosedAssessmentScore` access rule restriction is often used in conjunction with [disabling real-time grading](assessment.md#disabling-real-time-grading) and setting `"showClosedAssessment": false`. It may also be used in assessments with [manual grading](manualGrading.md), to hide the assessment grade until all students have been graded.
```